corehq/apps/sms/models.py

```diff
@@ -605,13 +606,17 @@ def __init__(self, owner):
 
     @property
     def phone_number(self):
-        return self.user_link.channel_id
+        return self.messaging_channel
```

Comment:
Is it possible to update calls to `phone_number` to just use `messaging_channel`?

Reply:
That is there for compatibility with the messaging code (this is meant to be equivalent to the `PhoneNumber` model used for sms, and was originally going to inherit from a base class to ensure compatibility but having a model and simple class inherit from the same base proved to be challenging). Messaging code references the phone number `phone_number` property consistently.